### ISSUE TITLE
[Enhancement] Publish hash or publish app on Gplay

### ISSUE THREAD

**Issue by ghost** [davidtakac/bura#67]
I really like this app, but the only way to install it is through sideloading, which breaks the AOSP TOFU security model. 

Would you be open to publishing the signing certificate hash on a separate website so that one doesn't have to blindly trust the GitHub release? 

That way [AppVerifier](https://github.com/soupslurpr/AppVerifier) could be used to verify the APK before installing and not have to blindly trust an app that has network permissions. 

Publishing on Gplay or Accrescent, as suggested in #21 would also fix this, Accrescent is still in whitelist only stage at the time of writing though, and obviously not the best option in terms of usability for devs.

**Comment by davidtakac** [comment]
> Would you be open to publishing the signing certificate hash on a separate website so that one doesn't have to blindly trust the GitHub release?

No. You do not have to blindly trust the GitHub release because it is automatically and [reproducibly](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14974#suggested) generated by GitHub actions from source when a Git tag is pushed. 

> Publishing on Gplay or Accrescent

Publishing to Google Play is not planned. I'll revisit Accrescent when it's out of alpha, see #21

**Comment by ghost** [comment]
@davidtakac, the issue is that there is no way to currently verify the app on-device to make sure you're not getting a malicious copy. Publishing the signing certificate hash on another website, for example a social media post would make it possible to verify it using AppVerifier.
